content/references/protocol-reference/transactions/transaction-types/xchainaccountcreatecommit.md

```diff
@@ -0,0 +1,65 @@
+---
+html: xchainaccountcreatecommit.html 
+parent: transaction-types.html
+blurb: Create a bridge between two chains.
+labels:
+  - Interoperability
+status: not_enabled
+---
+# XChainAccountCreateCommit
+
+The `XChainAccountCreateCommit` transaction defines a new cross-chain bridge entrance on one of the chains that the bridge connects. It includes information about the type of tokens being exchanged. To fully set up a bridge, this transaction must be executed on both chains, alongside setting up witness servers.
+
+The complete production-grade setup would also include a `SignerListSet` transaction on the two door accounts for the witnesses’ signing keys, as well as disabling the door accounts’ master key. This would ensure that the funds are truly in control of the witness servers.
+
+## Example {{currentpage.name}} JSON
+
+
+```json
+{
+  "Account": "rwEqJ2UaQHe7jihxGqmx6J4xdbGiiyMaGa",
+  "Destination": "rD323VyRjgzzhY4bFpo44rmyh2neB5d8Mo",
+  "TransactionType": "XChainAccountCreateCommit",
+  "Amount": "20000000",
+  "SignatureReward": "100",
+  "XChainBridge": {
+    "LockingChainDoor": "rMAXACCrp3Y8PpswXcg3bKggHX76V3F8M4",
+    "LockingChainIssue": {
+      "currency": "XRP"
+    },
+    "IssuingChainDoor": "rHb9CJAWyB4rj91VRWn96DkukG4bwdtyTh",
+    "IssuingChainIssue": {
+      "currency": "XRP"
+    }
+  }
+}
+```
+
+
+{% include '_snippets/tx-fields-intro.md' %}
+
+| Field         | JSON Type           | [Internal Type][] | Description        |
+|:--------------|:--------------------|:------------------|:-------------------|
+| `XChainBridge`| String | Object | _Required_ The XChainBridge stanza represents the bridge for which the witness is attesting transactions. |
+| `LockingChainDoor` | String | AccountID | The door account on the locking chain. |
+| `LockingChainIssue` | String | Token | The token that is bridged on the locking chain. |
+| `IssuingChainDoor` | String  |  AccountID | The door account on the issuing chain. |
+| `IssuingChainIssue` | String | Token | The token that is bridged on the issuing chain. |
+| `SignatureReward`  | Number  | Token |  _Required_ The total amount, in XRP, to be rewarded for providing a signature for cross-chain transfer or for signing for the cross-chain reward. This amount will be split among the signers. |
```

Comment:
Rather than putting _Required_ or _Optional_ in italics (previously I was also using parentheses to set it apart from the rest of the sentence), I've been trying to move to a "Required?" (Yes/No) column in all the references.

I suggest adding a **Required?** column to this table like you see on [AMMDeposit](https://xrpl.org/ammdeposit.html) for example.